src/geventhttpclient/useragent.py

```diff
@@ -365,7 +365,7 @@ def urlopen(self, url, method='GET', response_codes=valid_response_codes,
                     e.request = req
                     e.http_log = self._conversation_str(req.url, resp, payload=req.payload)
                     resp.release()
```

Comment:
What about this `release`?

Reply:
Anyways, thanks for all the contributed fixes so far. This package is looking for an active maintainer, in case you're interested.